include/DgmOctree.h

```diff
@@ -1250,6 +1253,25 @@ namespace CCCoreLib
 			\return the index of the cell (or 'm_numberOfProjectedPoints' if none found)
 		**/
 		unsigned getCellIndex(CellCode truncatedCellCode, unsigned char bitDec, unsigned begin, unsigned end) const;
+
+#ifdef ENABLE_MT_OCTREE
+		/*** FOR THE MULTI THREADING WRAPPER ***/
+		struct octreeCellDesc
+		{
+			DgmOctree::CellCode truncatedCode;
+			unsigned i1, i2;
+			unsigned char level;
+		};
+
+		DgmOctree* s_octree_MT = nullptr;
```

Comment:
if they are not static anymore, we should remove the 's_' prefix at the beginning. It may be worthwhile to regroup all these in a struct also (something like 'multiThreadXXX' (not super inspired for what XXX should be :D) and then remove the '_MT' suffix as well.